hardhat.config.ts

```diff
@@ -44,56 +45,53 @@ export default {
   zksolc: {
     version: "latest",
     settings: {
-      forceEvmla: true,
+      forceEvmla: false,
```

Comment:
Oh interesting, why `false`?

Reply:
The latest on main makes use of `^0.8.0`, we want to avoid using this flag so there is no change to their existing contracts that use recursion.